tests/api/test_get_resource.py

```diff
@@ -0,0 +1,486 @@
+# SPDX-License-Identifier: GPL-3.0+
+
+from __future__ import unicode_literals
+import json
+from datetime import datetime
+import pytest
+
+from purview.models.user import User
+from purview.models.bugzilla import BugzillaBug
+from purview.models.distgit import DistGitCommit, DistGitPush, DistGitBranch, DistGitRepo
+from purview.models.errata import Advisory, AdvisoryState
+from purview.models.koji import KojiBuild, KojiTask, KojiTag
+from purview.models.freshmaker import FreshmakerEvent, ContainerBuilds
+
+
+@pytest.mark.parametrize('resource,uid,expected', [
+    ('bugzillabug', '12345', {
+        'assignee': {
+            'email': 'matt.prahl@domain.local',
+            'name': None,
+            'username': 'mprahl'
+        },
+        'attached_advisories': [{
+            'actual_ship_date': '2017-08-01T15:43:51+00:00',
+            'advisory_name': 'RHBA-2017:2251-02',
+            'content_types': ['docker'],
+            'created_at': '2017-04-03T14:47:23+00:00',
+            'id': '27825',
+            'issue_date': '2017-08-01T05:59:34+00:00',
+            'product_name': 'Red Hat Enterprise Linux',
+            'product_short_name': 'RHEL',
+            'release_date': None,
+            'security_impact': 'None',
+            'security_sla': None,
+            'state': 'SHIPPED_LIVE',
+            'status_time': '2017-08-01T15:43:51+00:00',
+            'synopsis': 'cifs-utils bug fix update',
+            'type': 'RHBA',
+            'update_date': '2017-08-01T07:16:00+00:00',
+            'updated_at': '2017-08-01T15:43:51+00:00'
+        }],
+        'classification': 'Red Hat',
+        'creation_time': '2017-04-02T19:39:06+00:00',
+        'id': '12345',
+        'modified_time': '2018-02-07T19:30:47+00:00',
+        'priority': 'high',
+        'product_name': 'Red Hat Enterprise Linux',
+        'product_version': '7.5',
+        'qa_contact': {
+            'email': 'jsmith@domain.local',
+            'name': None,
+            'username': 'jsmith'
+        },
+        'related_by_commits': [],
+        'reporter': {
+            'email': 'tom.brady@domain.local',
+            'name': None,
+            'username': 'tbrady'},
```

Comment:
Can you use the same style as you did with `qa_contact` above, where the last curly brace is on a new line? Consistency of style is desired is all.